packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -192,66 +192,65 @@ export default function VisualEditor( { styles } ) {
 	}, [ isTemplateMode, themeSupportsLayout, contentSize, wideSize ] );
 
 	return (
-		<div
+		<BlockTools
```

Comment:
For me if this needs to be a parent of the block editor, it needs to be documented properly in the block-editor package and the documentation we have to build third-party block editors.

That said, the naming is not great at all, shouldn't be `BlockEditorWrapper` or something like that?  Also, if it's a wrapper, why not just bundle it in `BlockEditorProvider`

Reply:
I don't understand the question. My main motivation here is that it seems we're requiring two wrappers where we can require just one. Unless I'm not understanding properly both these components should wrap the editor canvas? Maybe more granularity is needed, but it's not clear from the user's perspective where to put each one of these components.